packages/react/src/themes/teams-dark/siteVariables.ts

```diff
@@ -1,34 +1,34 @@
-import { white, green } from '../teams/siteVariables'
+import { colors, naturalColors } from '../teams/siteVariables'
 
 //
 // COLORS
 //
-export const black = '#201f1f'
+export const black = '#201f1f' // no mapping color
 
-export const gray02 = '#c8c6c4'
-export const gray03 = '#b3b0ad'
-export const gray04 = '#8a8886'
-export const gray06 = '#605e5c'
-export const gray08 = '#484644'
-export const gray09 = '#3b3a39'
-export const gray10 = '#323130'
-export const gray14 = '#292828'
+export const gray02 = '#c8c6c4' // light theme gray06
+export const gray03 = '#b3b0ad' // no mapping color
+export const gray04 = '#8a8886' // no mapping color
+export const gray06 = '#605e5c' // light theme gray03
+export const gray08 = '#484644' // light theme gray02
+export const gray09 = '#3b3a39' // no mapping color
+export const gray10 = '#323130' // no mapping color
+export const gray14 = '#292828' // no mapping color
```

Comment:
It feels like `gray*` and `brand*` there should extracted as part of color pallete

Reply:
We have grey in the color palette, but the color are not matching these one, after the updates of the color palette, we may be able to reduce them. For the brand colors, currently I am mapping them to the primary, but we may change this to brand in the future.